.gitmodules

```diff
@@ -5,7 +5,7 @@
         ignore = all
 [submodule "sorc/MPAS-Model"]
 	path = sorc/MPAS-Model
-	url = https://github.com/MPAS-Dev/MPAS-Model
+	url = https://github.com/NOAA-GSL/MPAS-Model
         branch = master
```

Comment:
Change the branch name to match what we want

Reply:
Done